### ISSUE TITLE
Feat: Allow customization of "Elsewhere" node links

### ISSUE THREAD

**Issue by vidplace7** [MeshAddicts/meshinfo#95]
In the Node view, there are links to "Elsewhere".

Example:
https://info.mtnme.sh/node_6eb383a5.html

> Elsewhere
> [Armooo's MeshView](https://meshview.armooo.net/packet_list/1857258405)
> [Bay Mesh Explorer](https://app.bayme.sh/node/6eb383a5)
> [Liam's Map](https://meshtastic.liamcottle.net/?node_id=1857258405)
> [MeshMap](https://meshmap.net/#1857258405)

Currently these links are hardcoded in [frontend/src/pages/Node.tsx](https://github.com/MeshAddicts/meshinfo/blob/develop/frontend/src/pages/Node.tsx) to values that are unique to the Bay Area.
Please allow these links to be customized in the config.

**Comment by kevinelliott** [comment]
@vidplace7 You are right! This is definitely something worth doing. In our "next UI" branch we have removed them from the nav bar entirely, and will allow them to be customized on the node detail.

Additionally, we'll be creating a new dedicated page for centralizing general Meshtastic tooling and external links (and this page will be customizable and/or disable-able).
